pkg/cli/environment.go

```diff
@@ -44,6 +44,9 @@ const defaultMaxHistory = 10
 // defaultBurstLimit sets the default client-side throttling limit
 const defaultBurstLimit = 100
 
+// defaultQPS sets the default QPS value to 0 to disable the feature unless specified
```

Comment:
Specifying 0 defers the value being set by the implementor and instead QPS is set by the client-go library which sets a value of 5

Reply:
I have updated the comment to reflect this, following @mattfarina suggestion this is only in the comment and have not explicitly said 5 since this is an externally set dependency that may be tuned in that library over time.